File: PROJECT/Models/DTOs/Internal/Actions/ActionHistoryDTO.cs
namespace Models.DTOs.Internal.Actions
{
    public class ActionHistoryDTO
    {
        public int Id { get; set; }
        public int ActionId { get; set; }
        public int ActionName { get; set; }
        public decimal ActionExpense { get; set; }
        public int UserId { get; set; }
        public int OrderId { get; set; }
        public DateTime Date { get; set; }
    }
}

File: PROJECT/FRONTEND/Common/DTOs/ActionHistoryDTO.ts


export class ActionHistoryDTO { 
    constructor(obj:Partial<ActionHistoryDTO>) {
        Object.assign(this,obj);
    }
    
    public id: number;
    public actionId: number;
    public actionName: number;
    public actionExpense: number;
    public userId: number;
    public orderId: number;
    public date: Date;
}
